test/test-script.js

```diff
@@ -433,55 +437,51 @@ describe('FastZombie', function() {
   });
 
   it('should have health', function() {
-    var charger = new FastZombie(30, 10, 25);
     charger.health.should.equal(30);
   });
 
   it('should have a strength attribute', function() {
-    var charger = new FastZombie(30, 10, 25);
     charger.strength.should.equal(10);
   });
 
   it('should have a speed attribute', function() {
-    var charger = new FastZombie(30, 10, 25);
     charger.speed.should.equal(25);
   });
 
   it('should have a private maxHealth variable', function() {
-    var charger = new FastZombie(30, 10, 25);
     expect(charger.maxHealth).to.be.undefined;
   });
 
   it('should be alive', function() {
-    var charger = new FastZombie(30, 10, 25);
     charger.isAlive.should.equal(true);
   });
 
   it('should have a speedMultiplier attribute', function() {
-    var charger = new FastZombie(30, 10, 25, 2);
     charger.speedMultiplier.should.equal(2);
   });
 
   it('should have a buffSpeed method', function() {
-    var charger = new FastZombie(30, 10, 25, 2);
     charger.buffSpeed.should.exist;
   });
 
   it('should multiply zombies speed by speedMultiplier when buffSpeed called', function() {
-    var charger = new FastZombie(30, 10, 25, 2);
     var buffedSpeed = charger.speed * charger.speedMultiplier;
     charger.buffSpeed();
     charger.speed.should.equal(buffedSpeed);
   })
 }); // end FastZombie specs
 
+var tank;
```

Comment:
`var tank`; should go inside `describe('StrongZombie'...`
just before `beforeEach`